docs/user/dashboard.asciidoc

```diff
@@ -137,26 +152,21 @@ tools. To create a short URL, you must have write access to {kib}.
 === Import and export dashboards
```

Comment:
I know its not part of your PR, but maybe the import stuff should go under Save a dashboard and not have its own heading.  Because the link goes to saved objects.